dsalgo/Binary_Search.py

```diff
@@ -0,0 +1,31 @@
+# Returns index of x in the list/array if present, else -1 
+def binarySearch(arr, l, r, x): 
```

Comment:
use oop.
add docstrings for each function.
you have to write option for algorithm in the class